client/model.go

```diff
@@ -584,6 +585,16 @@ func (ps *PolicySet) AddPolicyFromReader(r io.Reader) *PolicySet {
 	return ps
 }
 
+func (ps *PolicySet) AddPolicyFromZipFile(file *zip.ReadCloser, path string) (*PolicySet, error) {
```

Comment:
I think it'd be rare to want to pick just a single file from a zip file. So, we could just simplify this to `AddFromZipArchive` with just the path to the zip file as the argument. 

If someone wants to add a single file from a zip, they can open the zip file themselves, locate the file they want and then call `AddPolicyFromReader` on the policyset.